src/Program.cs

```diff
@@ -156,9 +104,20 @@ private static int Run(RunOptions opts)
                 project.Save();
             }
 
+            // Executes the command to download the version if it doesn't exist
+            if (Pull(new PullOptions()
+            {
+                Version = project.Runtime.Version,
+                Pro = project.Runtime.Tier == ServerTier.Pro,
+                Platform = project.Runtime.Platform,
+                Force = false
+            }) != 0)
+            {
+                Console.Error.WriteLine(Output.Red("An error occured while trying to download the version required, exiting New"));
```

Comment:
Similar to the above comment, lets trim 'exiting New' and change the return code!

Reply:
I've definitely used the wrong exit code a few times in this utility!